MinVer/build/MinVer.targets

```diff
@@ -12,21 +12,25 @@
 
   <Target Name="MinVer_GetVersion" Condition=" '$(MinVerVersion)' == '' ">
     <Error Condition="'$(MinVerVerbose)' != '' And '$(MinVerVerbose)' != 'true' And '$(MinVerVerbose)' != 'false'" Code="MINVER0003" Text="MinVerVerbose/MINVER_VERBOSE value '$(MinVerVerbose)' cannot be converted to a Boolean value." />
-    <PropertyGroup>
-      <MinVerVerboseOption Condition="'$(MinVerVerbose)' == 'true'"> --verbose</MinVerVerboseOption>
-    </PropertyGroup>
-    <Exec Command="dotnet &quot;$(MSBuildThisFileDirectory)../MinVer/MinVer.Cli.dll&quot; --build-metadata &quot;$(MINVER_BUILD_METADATA)&quot; --path &quot;$(MSBuildProjectDirectory)&quot; --major-minor &quot;$(MinVerMajorMinor)&quot; --tag-prefix &quot;$(MinVerTagPrefix)&quot;$(MinVerVerboseOption)" ConsoleToMSBuild="true">
-      <Output TaskParameter="ConsoleOutput" ItemName="MinVerConsoleOutputItems" />
+    <ItemGroup>
+      <MinVerOptions Include="--build-metadata &quot;$(MINVER_BUILD_METADATA)&quot;" />
+      <MinVerOptions Include="--major-minor &quot;$(MinVerMajorMinor)&quot;" />
+      <MinVerOptions Include="--path &quot;$(MSBuildProjectDirectory)&quot;" />
+      <MinVerOptions Include="--tag-prefix &quot;$(MinVerTagPrefix)&quot;" />
+      <MinVerOptions Include="--verbose" Condition="'$(MinVerVerbose)' == 'true'" />
+    </ItemGroup>
+    <Exec Command="dotnet &quot;$(MSBuildThisFileDirectory)../MinVer/MinVer.Cli.dll&quot; @(MinVerOptions->'%(Identity)', ' ')" ConsoleToMSBuild="true">

```

Comment:
@bording do you have an opinion on this change? It's more lines, but it gets rid of `MinVerVerboseOption` and the 212 character options expression, which also had to have a placeholder for `MinVerVerboseOption`, and that had to include it's leading space separator, which always felt a bit ugly.

Reply:
I like it! Much cleaner, and it would be easier to add new options in the future that way.